sbc_compassion/models/import_letters_history.py

```diff
@@ -233,57 +155,62 @@ def onchange_config(self):
             for field, val in list(config.get_correspondence_metadata().items()):
                 setattr(self, field, val)
 
-    ##########################################################################
-    #                             PRIVATE METHODS                            #
-    ##########################################################################
+    def local_generator(self):
+        d = set(self.data)
```

Comment:
Please add a comment explaining this method and what it's used for. Should it be private (`local_generator`)?

Reply:
Specify in the comments the return type